Cargo.toml

```diff
@@ -21,11 +21,13 @@ maintenance = { status = "experimental" }
 
 [features]
 sanitize = ['crossbeam-epoch/sanitize']
+serialize = ["serde"]
```

Comment:
I think nowadays people just don't explicitly name this feature. By declaring the dependency as optional, people can opt in by naming `serde` as a feature, even though it isn't explicitly listed. See [`hashbrown/Cargo.toml`](https://github.com/rust-lang/hashbrown/blob/master/Cargo.toml).

Reply:
Makes sense. I'll work on this.